### DISCUSSION TITLE
/isearch not being displayed

### DISCUSSION THREAD

**Discussion by royalleh** [NadyaNayme/TidyChat#91] [Q&A]
For some reason my /isearch does not get displayed in chat even though it shows it as enabled.

![image](https://github.com/user-attachments/assets/c6bf871f-2f77-47e3-99da-3cf10a502a89)

when running /isearch guile without the debug mode on it doesnt display anything. When disabling Tidy Chat the /isearch displays as normal. I am not sure why 

![image](https://github.com/user-attachments/assets/e1f68432-50ee-4a0a-9bd4-620beeeb8091)
![image](https://github.com/user-attachments/assets/7e92c4f2-9686-49ec-8b39-2b6ff89b4bd6)
![image](https://github.com/user-attachments/assets/f7c36de0-08c7-453e-b404-1026babec6af)

**Comment by NadyaNayme** [comment]
Is this still an issue as of 2.0.10? The disabling for /? and /isearch wasn't working as intended. I fixed /? but forgot to test /isearch.

**Reply by royalleh** [reply]
Still happening as of 2.0.10.0

**Reply by NadyaNayme** [reply]
I'll take a look specifically for /isearch when I get a chance this week.
